lib/gpprmsya.gi

```diff
@@ -2447,6 +2447,42 @@ InstallMethod( RadicalGroup, "symmetric", true,
 InstallMethod( RadicalGroup, "alternating", true,
     [ IsNaturalAlternatingGroup and IsFinite], 0,RadicalSymmAlt);
 
+InstallMethod(NormalSubgroups,
+"for a perm group isomorphic to the symmetric group",
+[IsPermGroup and IsSymmetricGroup],
+function(G)
+  local pts, list;
+  pts := MovedPoints(G);
+  list := [Group(())];  # Trivial group
+  if Length(pts) = 4 then  # Klein 4-group
+    Add(list, Group((pts[1],pts[2])(pts[3],pts[4]),
+                    (pts[1],pts[3])(pts[2],pts[4])));
+  fi;
+  if Length(pts) > 2 then  # Alternating group
+    Add(list, AlternatingGroup(pts));
+  fi;
+  if Length(pts) > 1 then
+    Add(list, G);  # Symmetric group
+  fi;
+  return list;
+end);
+
+InstallMethod(NormalSubgroups,
+"for a perm group isomorphic to the alternating group",
+[IsPermGroup and IsAlternatingGroup],
```

Comment:
I guess something similar could also be installed for `IsSimpleGroup`... Not saying you should, just thinking out loud. The case with few moved points are mostly trivial to compute anyway.